index.html

```diff
@@ -0,0 +1,112 @@
+</head>
+
+<body>
+  <!--2. Adicione uma barra superior com um título-->
+  <header id="cabecalho">
+    <h1 id="titulo">Jornada Trybe</h1>
+    <nav>
+      <ul>
+        <li>Aprendizado na Trybe</li>
+        <li>Contatos</li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <!--3. Adicione uma foto sua à página  -->
+    <div class="imagem listas">
+      <img id="minha_foto" src="minha_foto.jpeg" alt="minha_foto">
+    </div>
+    <!--4. Adicione uma lista de lições aprendidas à página-->
+    <section class="listas">
+      <h2>Liçoes Aprendidas</h2>
+      <ol id="licoes_aprendidas">
+        <li>O que é Unix e Bash.</li>
+        <li>Principais comandos utilizados no Bash</li>
+        <li>O que é Git</li>
+        <li>O que é GitHub</li>
+        <li>Como sincronizar o Git com o GitHub</li>
+        <li>Entendendo a internet</li>
+        <li>Introdução sobre o HTML</li>
+        <li>Introdução sobre o CSS</li>
+        <li>HTML Semântico</li>
+        <li>Início ao conhecimento das soft skills</li>
+      </ol>
+    </section>
+    <section class="listas">
+      <!--5. Crie uma lista de lições que ainda deseja aprender para a página-->
+      <ul id="licoes_a_aprender">
+        <h2>Liçoes a Aprender</h2>
+        <li>Aprofundamento do HTML</li>
+        <li>Aprofundamento do CSS</li>
+        <li>JavaScript</li>
+        <li>Técnicas para melhorar o acesso do usuário</li>
+        <li>Técnicas para aumentar o engajamento da página</li>
+        <li>Trabalhar o design dos sites de forma ficar mais agradável</li>
+        <li>Formas eficientes de resolver "bugs" no site</li>
+        <li>Entender que outras aplicações são importantes também ter conhecimento ao fazer uma página, por exemplo, programas de edição de imagem</li>
+        <li>Aplicação Front-end</li>
+        <li>Aplicação Back-end</li>
+      </ul>
+    </section>
+    <br>
+    <!--8. Crie um artigo sobre seu aprendizado-->
+    <article class="caixa lado-direito">
+      <section>
+        <h2>Meu aprendizado na Trybe</h2>
+        <p>A Trybe apresenta um jeito único de fazer a gente aprender, aqui nós não nos desenvolvemos somente para seremprogramadores, mas sim, tudo que um verdadeiro profissional precisa possuir, como resiliência, respeito epreocupação pelo próximo. Vale afirmar também, que a ela nos fornece um material incrível, bem detalhado e comum nível além do básico. Além disso, conta com uma equipe de funcionários sensacional,que prestam apoio para todos independente do problema.</p>
+      </section>
+    </article>
+    <!--9. Crie uma seção que conta uma passagem sobre seu aprendizado-->
+    <section class="caixa lado-esquerdo">
+      <aside>
+        <h2>Uma passagem na Trybe</h2>
+        <p> O meu primeiro projeto na Trybe, se chamou "lessons-leaned". Neste, fui responsável por montar um site do zero, seguindo diversas regras que me desafiava a todo instante. Se você quiser enfrentar isso também, acesse o site da Trybe e se inscrevendo. </p>
```

Comment:
@GabrielFMPinheiro acredito que tenha um erro tipográfico neste item. 
```suggestion
        <p> O meu primeiro projeto na Trybe, se chamou "lessons-learned". Neste, fui responsável por montar um site do zero, seguindo diversas regras que me desafiavam a todo instante. Se você quiser enfrentar isso também, acesse o site da Trybe e se inscrevendo. </p>
```

Será que é possível incluir um link pro site da Trybe? Ou sai do requisito?

Reply:
Eu tive que alterar um pouco o texto, mas conseguir colocar o link. Obrigado!